protobuf-kafkaconnect-converter/pom.xml

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this create a uber jar like other converters?

Reply:
Yes we have the `maven-shade-plugin` to generate the uber jar